src/ontology/mondo-edit.obo

```diff
@@ -7619,8 +7619,6 @@ intersection_of: disease_has_location UBERON:0002358 ! peritoneum
 id: MONDO:0000651
 name: thoracic disease
 def: "A non-neoplastic or neoplastic disorder that affects the thorax and/or the organs of the thoracic cavity. Representative examples include pleural infection, mediastinitis, thymoma, mediastinal lymphoma, and pleural mesothelioma." [NCIT:C35742]
-comment: Reason: grouping class. Term to consider: none
```

Comment:
did you mean to remove this obs tag? Do we no longer want to obsolete this term?